pkg/server/bootstrap/identity.go

```diff
@@ -277,17 +277,31 @@ func decorateWildcardPathsWithResourceIdentities(urlPath string, ids *identities
 		return urlPath, nil
 	}
 
-	if strings.Contains(comps[5], ":") {
-		return urlPath, nil
+	// It's possible the incoming request already has an identity specified. Make sure we exclude that when
+	// determining the resource in question.
+	parts := strings.SplitN(comps[5], ":", 2)
+	if len(parts) == 0 {
```

Comment:
can this ever happen?

Reply:
Probably not, but can't hurt to have the code here?